style.css

```diff
@@ -316,5 +316,145 @@ a {
     margin: 8px 20px;
   }
 
+  #contact {
+    padding: 80px 0 0 0;
+    background-color: #fff;
+  }
+
+  footer {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    border-radius: 70px 0 0 0;
+    background-color: #6070ff;
+  }
+
+  footer h2 {
+    margin-top: 103px;
+    margin-bottom: 6px;
+    color: #fff;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+  }
+
+  #para {
+    color: #ebebff;
+    margin: 6px 24px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+  }
+
+  form {
+    display: flex;
+    flex-direction: column;
+    width: 90%;
+  }
+
+  .name {
+    border: 1px solid #cfd8dc;
+    border-radius: 5px;
+    background: #fff;
+    margin: 87px 0 10px 0;
+    padding: 10px;
+    height: 80%;
+    width: 95%;
+  }
+
+  .email {
+    border: 1px solid #cfd8dc;
+    border-radius: 5px;
+    background: #fff;
+    padding: 10px;
+    margin: 10px 0;
+    height: 80%;
+    width: 95%;
+  }
+
+  .textarea {
+    border: 1px solid #cfd8dc;
+    border-radius: 5px;
+    background: #fff;
+    margin: 12px 0;
+    width: 100%;
+  }
+
+  .button {
+    margin: 12px 0 14px 0;
+    padding: 12px 16px;
+    width: 30%;
+    height: 10%;
+    background: #fff;
+    border-radius: 8px;
+    color: #6070ff;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    letter-spacing: 0.03em;
+    border: 1px solid #cfd8dc;
+  }
```

Comment:
- [ ] Kindly fix the width of the button to improve your UI design.
<img width="81" alt="image" src="https://user-images.githubusercontent.com/77190203/155185773-087c07df-60b5-4d86-84b1-10816c7d7bed.png">